BLOG-POST-TEMPLATE.md

```diff
@@ -61,7 +62,20 @@ the blog post.
 Once you have the file created, you can paste the contents of this template into
 the new file and begin to modify it.
 
-## Blog Head
+## Scheduling a blog post for the future
+
+To schedule a blog post to be released on a particular date, simply create that
+blog post with a date in the future. This should be reflected both in the
+content directories in which the post and any static content are stored, and in
+the blog head metadata ([see below](#blog-head)).
```

Comment:
Are we building the blog on a schedule or is it only on merges to `main`? If it's the latter, then we'd need to trigger a build manually on the days that new content is scheduled to be published.